### PR TITLE
Password reset Cleaner text

### LINKED ISSUE #6229: Password reset confirmed also at non-existing e-mail address
When asking for a password reset, a notification pops up, that an e-mail has been sent with further instructions, even with non-existing e-mail addresses. We would expect saying that there is no user associated to that e-mail address.
Furthermore, if you repeat this, the "Password reset" button is duplicated (see screenshot):
bildschirmfoto_2017-03-03_11-14-06

Rocket.Chat Version: 0.52

### PR DESCRIPTION
**Pull request by MartinSchoeler**
@RocketChat/core 

Related to https://github.com/RocketChat/Rocket.Chat/issues/6229#issuecomment-284205677

screenshot from 2017-03-10 14-24-44

### CODE REVIEW

File: server/methods/sendForgotPasswordEmail.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ Meteor.methods({
 		email = email.trim();
 
 		const user = RocketChat.models.Users.findOneByEmailAddress(email);
+		if (!user) {
```
Comment (karlprieb) [inline server/methods/sendForgotPasswordEmail.js]: I think you can use an `else` in the `if` below, it will check for `user` once one time.

Comment (MartinSchoeler) [inline server/methods/sendForgotPasswordEmail.js]: That shouldn’t be there, my bad.

**Comment by alexbrazier** [comment]
As an alternative to this, some websites send an email to the address you enter either giving you instructions to reset your password if the account exists, or sends you an email saying that you haven't registered with the email.
I think this is a good way of doing it because it means you are not waiting for an email that will never come if you don't have an account, and it is not a security issue because you will only be told if an account exists for an email if you own the email.

E.g.
image

**Comment by rodrigok** [comment]
I liked the @alexbrazier idea

**Comment by MartinSchoeler** [comment]
@karlprieb @sampaiodiego Could you do a review of this pr again? I have already did the changes

**Comment by rodrigok** [comment]
@sampaiodiego What do you think?

**Pull request merged**